packages/atomic/src/components/recommendations/atomic-recs-result/atomic-recs-result.tsx

```diff
@@ -161,8 +161,9 @@ export class AtomicRecsResult {
     }
     return (
       // deepcode ignore ReactSetInnerHtml: This is not React code
-      <Host class={resultComponentClass}>
+      <Host exportparts="result-list-grid-content" class={resultComponentClass}>
         <div
+          part="result-list-grid-content"
```

Comment:
The result root is technically already style-able by adding a `style` tag to its template and using class selectors. It's even possible to target specific display modes. See https://github.com/coveo/ui-kit/blob/d1f650d2097b8873c69c15910aff2547ff5d4aee/packages/atomic/src/pages/index.html#L256-L259

I don't think we should add this unless we decide that we also need it for `atomic-result`/`atomic-result-list`, which I don't think we do.